format/FlightSql.proto

```diff
@@ -1778,6 +1789,44 @@ message CommandPreparedStatementUpdate {
   bytes prepared_statement_handle = 1;
 }
 
+  /*
+   * Represents a bulk ingestion request. Used in the command member of FlightDescriptor
+   * for the the RPC call DoPut to cause the server load the contents of the stream's
+   * FlightData into the target destination.
+   */
+message CommandStatementIngest {
+  option (experimental) = true;
+
+  // Describes the behavior for loading bulk data.
+  enum IngestMode {
+    // Ingestion behavior unspecified.
+    INGEST_MODE_UNSPECIFIED = 0;
+    // Create the target table. Fail if the target table already exists.
+    INGEST_MODE_CREATE = 1;
+    // Append to an existing target table. Fail if the target table does not exist.
+    INGEST_MODE_APPEND = 2;
+    // Drop the target table if it exists. Then follow INGEST_MODE_CREATE behavior.
+    INGEST_MODE_REPLACE = 3;
+    // Create the target table if it does not exist. Then follow INGEST_MODE_APPEND behavior.
+    INGEST_MODE_CREATE_APPEND = 4;
+  }
+
+  // The ingestion behavior.
+  IngestMode mode = 1;
+  // The table to load data into.
+  string target_table = 2;
+  // The db_schema of the target_table to load data into. If unset, a backend-specific default may be used.
+  optional string target_schema = 3;
+  // The catalog of the target_table to load data into. If unset, a backend-specific default may be used.
+  optional string target_catalog = 4;
+  // Use a temporary table for target_table.
```

Comment:
`target_` prefix may be redundant.

Reply:
I was loosely following the existing conventions in [StatementOptions](https://github.com/apache/arrow-adbc/blob/main/python/adbc_driver_manager/adbc_driver_manager/__init__.py#L113) which uses the `target_` prefix. I agree though that it doesn't actually improve clarity here.

Reply:
I removed the `target_` prefix as it doesn't add much information in this context.